api/api/test/test_validator.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python
+# Copyright (C) 2015-2019, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import pytest
+
+from api import validator
+
+
+@pytest.mark.parametrize('exp, regex_name', [
```

Comment:
I would sort the parametrize list, grouping by similar regex_name and exp.

Reply:
done!